charts/sn-platform/values.yaml

```diff
@@ -1099,6 +1107,14 @@ broker:
     # annotations that will be added on the PulsarBroker CR only.
     annotations: {}
 
+  # The field autoScaling will track the average metrics usage of the Pulsar nodes and adjusts the nodes to keep at the target metrics usage level.
+  # If the average metrics usage for nodes is over the target, scaling controller will scale out to bring more nodes and if the average metrics for the nodes is less than the target, scaling controller will downscale nodes to save resources.
+  autoScaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 4
+    metrics: {}
```

Comment:
what are the available metrics here?

Reply:
It would be great if there is a list of the metric names that can be used for pulsar auto-scaling.

Reply:
> what are the available metrics here?
It depends on the metrics backend service used. If users deploy the Metrics Server then only resource metrics could be used. But with prometheus and prometheus adapter, all metrics from the prometheus could be used by the auto-scaling.

Reply:
The `cpu_usage` here is a  custom metric implemented by [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) other than the general prometheus metric,  the user should know the catalog of the supported metrics of broker autoScaling.